backend/packages/Upgrade/src/api/controllers/ExperimentController.ts

```diff
@@ -952,28 +953,23 @@ export class ExperimentController {
    */
 
   @Post()
+  @UseBefore(ValidateMoocletPolicyParametersMiddleware)
```

Comment:
This middleware will be going to get used in multiple end point?

Reply:
No it should just be here, is there a different way you'd recommend?